Add BoD contact link; remove Transition Group para

placer14 opened the pull request:
This change will:
- remove the Transition Group paragraph. 
- add an email link to contact the BoD via info@nationofmakers.us
- The Transition Group para is not added back in because it's included in `about/index.html` already. I've included a screengrab of the about page attached below.

Board of Directors Section now looks like:
![image](https://cloud.githubusercontent.com/assets/305332/24937761/e44eaf88-1f00-11e7-9bbc-a217b9bc7f3d.png)

About page looks like:
![image](https://cloud.githubusercontent.com/assets/305332/24937782/0b22e84a-1f01-11e7-93f6-b9bf5403bb3a.png)

placer14 committed:
Add BoD contact link; remove Transition Group para

placer14 committed:
Fix some minor spacing on BoD contact link

stevegt commented:
per slack Mike is changing "contact BoD" to "contact us" -- standing by for that

stevegt committed:
Change email description from "Board of Directors" to "Nation of Makers"

stevegt committed:
remove leading '/' from image filenames

If we avoid absolute paths we can use our forked repos for dev/qa:

https://username.github.io/nationofmakers.github.io/

mkenigson reviewed (approved):
lgtm
